package.json

```diff
@@ -59,6 +59,7 @@
         "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
     },
     "devDependencies": {
+        "@babel/core": "^7.0.0",
```

Comment:
I was getting peer dependency errors from this. Let me know if this is an issue

Reply:
No problem